generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -431,7 +431,7 @@ public class <%= asEntity(entityClass) %> implements Serializable {
         const fieldInJavaBeanMethod = fields[idx].fieldInJavaBeanMethod; _%>
 
     <%_ if (fieldTypeBlobContent !== 'text') { _%>
-    public <%= fieldType %> <% if (fieldType.toLowerCase() === 'boolean') { %>is<% } else { %>get<%_ } _%><%= fieldInJavaBeanMethod %>() {
+    public <%= fieldType %> get<%= fieldInJavaBeanMethod %>() {
```

Comment:
Instead of dropping the condition, remove `toLowerCase()`.
So if fieldType is primitive boolean, we will generate with `is`.

Reply:
According to the current generator, it will never be primitive, even set "required"  still is "Boolean" with annotation "@NotNull"

Reply:
Yep, that’s right.
Boolean primitive is useful due to checkbox without a default value would require the user to click twice to get false value.
I wanted to keep the template even if not used, blueprints could use this, but seems too much.